### ISSUE TITLE
Declutter labels in vector layers

### ISSUE THREAD

**Issue by brentfraser** [geomoose/gm3#572]
Labelling vector layers can lead to lots of label overlap making things hard to read.  OpenLayers 6 has an option on vector layers to do [automatic decluttering](https://openlayers.org/en/latest/examples/vector-label-decluttering.html).  GM should add the option to vector layers, something like the following in [vector.js](https://github.com/geomoose/gm3/blob/master/src/gm3/components/layers/vector.js#L294):

```
    const opts = {
        source,
        minResolution: mapSource.minresolution,
        maxResolution: mapSource.maxresolution,
        declutter: true
    };
```
or perhaps make it user-settable by adding a declutter attribute to the mapsource layer tag.

**Comment by theduckylittle** [comment]
@brentfraser - That option is already set:

```
https://github.com/geomoose/gm3/blob/master/src/gm3/components/layers/vector.js#L301
```

**Comment by brentfraser** [comment]
Ah yes it was included in [8d81393 ](https://github.com/geomoose/gm3/commit/8d8139344b0640da748464d5dd0c21d5c55866df#diff-67d316d08345ee95aa3871d6d9ed2ace)
